quests/2-auth-store/src/test.rs

```diff
@@ -142,7 +140,7 @@ fn test_contract_store() {
 /// as another smart contract. This is expected to fail since that method is
 /// only available to an `Account`, and not a `Contract`.
 #[test]
-#[should_panic(expected = "Status(ContractError(1))")] // We want this test to panic since it uses a forbidden function.
```

Comment:
Is it possible to have an `or` statement here? If they _do_ implement the function I would want the test to pass but my guess is it would fail atm?

Reply:
If users do implement a function that makes the test succeed, then they would also need to delete the `#[should_panic]`. I don't think that there is a way to make the should_panic conditional on the implementation or not of a function.

Reply:
Okay fair enough, but we should clue the user into what they need to add or comment out in the test once they reenable the function in the lib.rs file. We should include that in the test.rs file so it's obvious as this isn't the point of this quest to learn how to write or fix tests.